data-table/table.js

```diff
@@ -64,6 +64,16 @@ function addColumn(numberOfColumns) {
      * createEntryCell(). Maybe should also accept index as an argument?
      * TODO: Fill this out
      */
+    var index = numberOfColumns.cellIndex;
+    var tab = document.getElementById('_tableDiv_'); // get the table informatio 
```

Comment:
This probably won't be the eventual div id, but I expect it will be fixed later. Add a TODO.